client/control/components/Header.js

```diff
@@ -2,11 +2,13 @@ import React, { PropTypes as pt } from 'react';
 import { Link } from 'react-router';
 import Breadcrumbs from 'react-breadcrumbs';
 
+import absolutePath from '../../utils/absolute-path';
```

Comment:
I find some humor in using a relative import path to import something called "absolute-path". (this is obviously the right think, just amusing)